newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+
+import * as React from 'react';
+import { t, Trans } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+
+import { Card } from '@material-ui/core';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import CheckCircleIcon from '@material-ui/icons//CheckCircle';
+
+import { ResponsiveLineStackLayout } from '../../UI/Layout';
+import CardContent from '../../UI/Card/CardContent';
+import Text from '../../UI/Text';
+import { Column, LargeSpacer, Line, Spacer } from '../../UI/Grid';
+import IconButton from '../../UI/IconButton';
+import GDevelopThemeContext from '../../UI/Theme/ThemeContext';
+
+import Rating from './Rating';
+
+import {
+  updateComment,
+  type Comment,
+  type GameRatings,
+} from '../../Utils/GDevelopServices/Play';
+import { useDebounce } from '../../Utils/UseDebounce';
+import { type AuthenticatedUser } from '../../Profile/AuthenticatedUserContext';
+
+const styles = { textComment: { whiteSpace: 'pre-wrap' } };
+
+type Props = {
+  comment: Comment,
+  authenticatedUser: AuthenticatedUser,
+  onUpdateComment: () => ?Promise<void>,
+};
+
+const getRatings = (ratings: ?GameRatings) => {
+  if (!ratings) return null;
+  if (ratings.version === 1) {
+    return [
+      {
+        key: 'rating-sound',
+        label: <Trans>Sound</Trans>,
+        value: ratings.sound,
+      },
+      {
+        key: 'rating-visuals',
+        label: <Trans>Visuals</Trans>,
+        value: ratings.visuals,
+      },
+      { key: 'rating-fun', label: <Trans>Fun</Trans>, value: ratings.fun },
+      {
+        key: 'rating-ease-of-use',
+        label: <Trans>Ease of use</Trans>,
+        value: ratings.easeOfUse,
+      },
+    ];
+  }
+};
+
+const FeedbackCard = ({
+  comment,
+  authenticatedUser,
+  onUpdateComment,
+}: Props) => {
+  const { getAuthorizationHeader, profile } = authenticatedUser;
+  const ratings = getRatings(comment.ratings);
+  const theme = React.useContext(GDevelopThemeContext);
+
+  const [processed, setProcessed] = React.useState<?boolean>(null);
+
+  const isProcessed = processed === null ? !!comment.processedAt : processed;
```

Comment:
I feel like you can simplify this by:
- initialising the boolean `processed` with !!comment.processedAt
- and you can then always use `processed` instead of `isProcessed`